tests/db/test_bigchain_api.py

```diff
@@ -9,27 +9,6 @@
 
 class TestBigchainApi(object):
 
-    @pytest.mark.tendermint
-    def test_get_spent_with_double_inclusion_detected(self, b, alice):
```

Comment:
Could you please explain why this test was removed?

Reply:
because we don't have a double inclusion anymore. The test was supposed to test the exception `CriticalDoubleInclusion` that was removed in #2415. Now with the changes, everything that happens in this test, happens in some other tests too, so we don't need it anymore. If you have an idea how to test a double inclusion I'll gladly rewrite it :)

Reply:
I think this test should just work as it is i.e. the test tries to insert two transactions with the same transaction ids so it would make sense to keep this test to ensure the transactions are uniquely indexed by their ids.

Reply:
We still tests the `OperationError` in another test: https://github.com/bigchaindb/bigchaindb/blob/f12734237e4fb32a899e3e0554f10e39f618e069/tests/db/test_bigchain_api.py#L47
and the existing double spend errors in the test below: https://github.com/bigchaindb/bigchaindb/blob/f12734237e4fb32a899e3e0554f10e39f618e069/tests/db/test_bigchain_api.py#L16